src/main/scala/com/danielasfregola/twitter4s/http/serializers/CustomFormats.scala

```diff
@@ -40,6 +40,15 @@ private[twitter4s] case object LocalDateSerializer
         case date: LocalDate => JString(date.toString)
       }))
 
+private[twitter4s] case object ZonedDateTimeSerializer
```

Comment:
Well done! :D

Reply:
Thank's :D 
And by the way, do you know when you will create a new release that includes these changes ?

Reply:
I'll do my best to release within 2 weeks :)

Reply:
Nice, that sounds great !
Thanks for your time :)

Reply:
version 7.0 contains this change and it is on its way to maven central :)

Reply:
Wonderful 🎉
Thank's a lot !